app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -430,6 +429,16 @@ fun ConversationScreen(
     }
 }
 
+private fun conversationScreenOnBackButtonClick(
+    messageComposerViewModel: MessageComposerViewModel,
+    focusManager: FocusManager,
+    navigator: Navigator
+) {
+    messageComposerViewModel.sendTypingEvent(TypingIndicatorMode.STOPPED)
```

Comment:
thought: Not for this PR itself, but it would be best if the the `ViewModel` isn't commanded by the UI. _i.e._ instead of `viewModel.sendTypingEvent` it could be something like `viewModel.onNavigateBack` and then the ViewModel decides what should be done.